syntax = "proto3";

package user;

service UserService {
  rpc UserLogin (User) returns (ReplyToken) {}
  rpc UserSignup (User) returns (ReplySuccess) {}
}

message User{
  string username = 1;
  string password = 2;
}

message ReplyToken {
  string token = 1;
}

message ReplySuccess {
  string success = 1;
}